napari/utils/list/_model.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, basetype, iterable=(), lookup=None):
             removed=None,
             reordered=None,
             changed=None,
+            name=Event,
```

Comment:
btw, `Event` is the default when adding a `kwarg=None` to an `EmitterGroup`, so it's not necessary to import here (I know that's inconsistent across the codebase).  It gets assigned  [here](https://github.com/napari/napari/blob/59ed366e9d492a2389c451468fd8b9f96508b4e2/napari/utils/event.py#L724-L726)